app/assets/javascripts/modules/search-highlight.es6

```diff
@@ -0,0 +1,29 @@
+{
+  'use strict';
+
+  class SearchHighlight {
+    start(el) {
+      const term = this.getUrlParameter(encodeURIComponent('search[q]')),
+      regEx = new RegExp(`(${term})`, 'gi');
+
+      if (!term) {
+        return;
+      }
+
+      $(el).find('.js-allow-highlighting').each((index, td) => {
+        $(td).html($(td).html().replace(regEx, '<mark>$1</mark>'));
+      });
+    }
+
+    getUrlParameter(name) {
+      name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+
+      const regex = new RegExp(`[\\?&]${name}=([^&#]*)`),
+      results = regex.exec(location.search);
```

Comment:
Just for completion I've added a trello card for these sorts of things here: https://trello.com/c/mmTSxhYa/244-javascript-style-consistencies